### PR TITLE
Events Updates

### PR DESCRIPTION
**Pull request by kave**
### Changes
- Rename Events body field => Subheading 
- Switch render location of live/future body fields to under the map (new body location)
- Change date used for post preview from date published to actual event date. 

@kurtw @richaagarwal

### CODE REVIEW

File: cfgov/v1/models/learn_page.py [hunk 1]
```diff
@@ -122,7 +122,7 @@ class Meta:
 
 class EventPage(AbstractFilterPage):
     # General content fields
-    body = RichTextField(blank=True)
+    body = RichTextField('Sub-Heading',blank=True)
```
Comment (kurtw) [inline cfgov/v1/models/learn_page.py]: I really think renaming the field is better than changing the label. Either way still requires a migration and we can still remain consistent. _stopping now_

Comment (richaagarwal) [inline cfgov/v1/models/learn_page.py]: Why is there a hyphen between sub and heading?

Comment (kave) [inline cfgov/v1/models/learn_page.py]: @richaagarwal this is a good question. no good reason. i've removed it

**Comment by kave** [comment]
@richaagarwal ready for review again

**Comment by richaagarwal** [comment]
👍

**Pull request merged**